src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -196,20 +199,39 @@ && isNotificationEventValidToSendToOtherPartySubscription(otherParty.getOtherPar
         return otherPartySubscription;
     }
 
+    private boolean canSendBasedOnConfidentiality(String partyMember) {
+        if (!(DIRECTION_ISSUED.equals(getNotificationType()) || DIRECTION_ISSUED_WELSH.equals(getNotificationType()))) {
+            return true;
+        }
+
+        String confidentialityType = responseWrapper.getNewSscsCaseData().getConfidentialityType();
+        List<String> confidentialityPartyMembers = responseWrapper.getNewSscsCaseData().getConfidentialityPartyMembers();
+
+        if (isNull(confidentialityType)
+                || isNull(confidentialityPartyMembers)
+                || "general".equalsIgnoreCase(confidentialityType)) {
```

Comment:
Is there anywhere we can pull this string in from, instead of using the literal value? Could even declare it at top of class (again extensibility/reuse and avoids using literals in code).